includes/_nav.htm

```diff
@@ -36,7 +36,7 @@
         <a class="b-nav__apache nav__item nav__sub__item" href="http://www.apache.org/security/" target="_blank">security</a>
       </div>
 
-      <a class="btn" href="/downloads">download</a>
+      <!-- <a class="btn" href="/downloads">download</a> -->
```

Comment:
is this commenting intentional? i imagine we would want to keep the big ol' download button even if we have the header? having it clearly highlighted seems important

Reply:
yeah, it was intentional. i was trying to clean up the page since IMHO there are way too many headings on the left-hand nav. but i can see how removing this might be too much. i was using this page for inspiration (which does have a big ol' icon, but is definitely better design): http://mesos.apache.org/